.github/actions/cargo-semver-checks/action.yml

```
@@ -0,0 +1,28 @@
+name: "Run cargo semver-checks"
+description: "Install and run the cargo semver-checks tool"
+inputs:
+  crate:
+    required: true
+    description: "The crate to run `cargo semver-checks` on."
+runs:
+  using: "composite"
+  steps:
+    - run: wget -q -O- https://github.com/obi1kenobi/cargo-semver-checks/releases/download/v0.18.3/cargo-semver-checks-x86_64-unknown-linux-gnu.tar.gz | tar -xz -C ~/.cargo/bin
+      shell: bash
+
+    - name: Get released version
+      shell: bash
+      id: get-released-version
+      run: |
+        MAX_STABLE_VERSION=$(curl https://crates.io/api/v1/crates/${{ inputs.crate }} --silent | jq '.crate.max_stable_version')
+        echo "version=${MAX_STABLE_VERSION}" >> $GITHUB_OUTPUT
+
+    - uses: actions/cache@v3
+      with:
+        path: ${{ github.workspace }}/target/semver-checks/cache
+        key: semver-checks-cache-${{ inputs.crate }}-steps.get-released-version.outputs.version
```

```suggestion
        key: semver-checks-cache-${{ inputs.crate }}-${{ steps.get-released-version.outputs.version }}
```
Did you mean to evaluate this, perhaps?

---

I definitely did haha